staging/src/k8s.io/csi-translation-lib/plugins/aws_ebs.go

```diff
@@ -253,3 +259,30 @@ func KubernetesVolumeIDToEBSVolumeID(kubernetesID string) (string, error) {
 
 	return awsID, nil
 }
+
+func getAwsRegionFromZones(zones []string) (string, error) {
+	regions := sets.String{}
+	if len(zones) < 1 {
+		return "", fmt.Errorf("no zones specified")
+	}
+
+	// AWS zones can be in four forms:
```

Comment:
what about copying/using https://github.com/kubernetes/kubernetes/blob/f48972e671e5f338d0d6adfe6055aba4534569ad/staging/src/k8s.io/legacy-cloud-providers/aws/aws.go#L1240 ? seems simpler and less likely to break

Reply:
u r right it will break on wavelength zone .